recipes/bcolz-zipline/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "bcolz-zipline" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bcolz-zipline-{{ version }}.tar.gz
+  sha256: c09ef10714db585d3e22c54efaf825b007c6e413f37af7d2e2ef5a31700d1226
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - cython
+    - numpy
+    - toml
+    - pip
+    - setuptools
+    - setuptools_scm
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  commands:
+    - python -c "import bcolz; bcolz.test()"
+  requires:
+    - numpy
+    - setuptools
+
+about:
+  home: https://pypi.org/project/bcolz-zipline/
+  summary: Columnar and compressed data containers.
+  license: BSD-1-Clause
```

Pretty sure it's actually:
```suggestion
  license: BSD-3-Clause
```